templates/docs/examples/patterns/notifications/variants-dark.html

```diff
@@ -0,0 +1,178 @@
+{% extends "_layouts/examples.html" %}
+{% block title %}Notifications / Variants dark {% endblock %}
+
+{% block standalone_css %}patterns_notifications{% endblock %}
+
+{% block style %}
+<link rel="stylesheet" href="{{ versioned_static('build/css/standalone/patterns_icons-additional.css') }}" />
+{% endblock %}
+
+{% set is_dark = true %}
+{% block content %}
+<h5>Borderless</h5>
```

Comment:
As far as I understand this file is practically a copy of "light" theme "variants.html". The only difference is `is_dark` that sets the class on body. The content itself is identical.

So to avoid this duplication, I'd suggest extracting the contents of `{% block content %}` (the identical part) to a separate file (for example _variants.html), and import it in both `variants.html` and `variants-dark.html`.

Similar to how it's done here: https://github.com/canonical/vanilla-framework/blob/main/templates/docs/examples/brochure/25-75-nested.html

This way we don't duplicate the whole list of notifications (and can update them in one place in future).